decoder.go

```diff
@@ -43,6 +43,17 @@ func (d *Decoder) Decode(obj interface{}) error {
 	return d.r.Error
 }
 
+// Read reads the next Avro encoded value from its input and returns it as byte array.
```

Comment:
This is not what this does. This reads the current buffer with possibly multiple values and possibly multiple messages. It is not possible to read the next value without basically decoding it.

Reply:
Weirdly working via OCF perfectly.

If there is no way without decoding it, I should close this PR. Because there is no purpose of it.

Reply:
If you are continually reading this to call write in OCF, the block count is going to be wrong in the OCF block. Not sure it matters in your at all, but it is not correct.